src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    static int numberOfTasks = 0;
+    public Task (String description) {
+        this.description = description;
+        this.isDone = false;
+        numberOfTasks++;
+    }
+
+    public Task (String description, boolean isDone) {
+        this(description);
+        this.isDone = isDone;
+    }
+
+    public static String getTasksList(Task[] tasks){
+        String tasksList = "";
+        for (int i = 0; i < numberOfTasks; i++) {
+            tasksList +=  String.format("%3d. ", (i+1)) + tasks[i].toString() + "\n\t";
+        }
+        return tasksList;
+    }
```

Comment:
Should this function be stored outside? Or the class be changed to a TaskList which includes the array of tasks?